components/02-elements/divider/_divider.scss

```diff
@@ -0,0 +1,26 @@
+$divider__color--left : $color-primary !default;
+$divider__color--right: $gray-lighter !default;
+$divider__size        : 8px !default;
+.divider {
+    display: none;
+
+    @include mq($screen-m) {
+        display: flex;
+    }
+
+    &__left,
+    &__right {
+        border-top: $divider__size solid;
+        flex: 1;
+    }
+
+    &__left {
+        border-color: $divider__color--left;
+        flex-grow: 1;
+    }
+
+    &__right {
+        border-color: $divider__color--right;
+        flex-grow: 8;
+    }
+}
```

Comment:
Interesting point, I haven't thought about changing the size of a container, thanks ! Right now I have adjusted the proportion of colors inside this divider, so it's more like design. About using pseudo element, do you have any idea of how it could look like in code ? I don't want to use absolute positioning here.